### ISSUE TITLE
Instructions for adding a new module

### ISSUE THREAD

**Issue by seisman** [GenericMappingTools/gmt#3313]
After PR #3291 is merged, we should document what files should be added/updated when adding a new module.

**Comment by PaulWessel** [comment]
yes, will go in to a doc on adding new modules, adding supplements.  Just not have had time.

**Comment by PaulWessel** [comment]
Started, see #3317
